Dockerfile

```diff
@@ -1,41 +1,66 @@
-FROM ruby:3.0.4 as base
+# Use the official Ruby image from Docker Hub
+# https://hub.docker.com/_/ruby
 
-RUN curl -sL https://deb.nodesource.com/setup_16.x | bash -
-RUN apt-get update -qq && apt-get install -y build-essential \
-    libpq-dev nodejs  \
-    libqt5webkit5-dev dos2unix \
-    gstreamer1.0-plugins-base gstreamer1.0-tools gstreamer1.0-x
+# [START cloudrun_rails_base_image]
+# Pinning the OS to buster because the nodejs install script is buster-specific.
+# Be sure to update the nodejs install command if the base image OS is updated.
+FROM ruby:3.0-buster as base
```

Comment:
are we running 3.0.5 here with this?

Reply:
Yes.  I think they keep 'buster' up with the 'stable' version of ruby 3.0.x